frontend/src/lib/components/KeyboardHotkeys.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEventListener } from 'lib/hooks/useEventListener'
+import { Keys } from '~/types'
+
+export interface HotKeyInterface {
+    action: () => void
+    disabled?: boolean
+}
+
+export type HotKeys = Partial<Record<Keys, HotKeyInterface>>
+
+export function KeyboardHotkeys({ hotkeys }: { hotkeys: HotKeys }): null {
```

Comment:
Since this doesn't actually return any JSX element, should be a React hook instead

Reply:
Right! Will make that change

Reply:
As discussed on Zoom, reverting back to functional component instead of hook as it was causing some obscure React error.